scripts/production/homology_stats_rapid.py

```diff
@@ -171,6 +187,12 @@ def main() -> None:
             "reference_species": ref_species,
         }
 
+    if args.refcoll:
+        json_data["refcoll_info"] = {
+            "refdb_version": get_meta_value(rr_dbc, "refdb_version"),
+            "ref_coll": get_meta_value(rr_dbc, "ref_coll"),
+        }
```

Comment:
Could you be tempted to use a `dict` comprehension here?
```suggestion
        json_data["refcoll_info"] = {k: get_meta_value(rr_dbc, k) for k in ["refdb_version", "ref_coll"]}
```